From: Markus Elfring <elfring@users.sourceforge.net>
Date: Thu, 5 Oct 2017 13:16:51 +0200
Subject: KVM: PPC: Book3S HV: Delete an error message for a failed memory
 allocation in kvmppc_allocate_hpt()
Patch-mainline: v4.15-rc1
Git-commit: 9c7e53dc00a543478b46c6a90694a41be69a089d
References: bsc#1077761

Omit an extra message for a memory allocation failure in this function.

This issue was detected by using the Coccinelle software.

Signed-off-by: Markus Elfring <elfring@users.sourceforge.net>
Signed-off-by: Paul Mackerras <paulus@ozlabs.org>
Acked-by: Alexander Graf <agraf@suse.de>
---
 arch/powerpc/kvm/book3s_64_mmu_hv.c |    1 -
 1 file changed, 1 deletion(-)

--- a/arch/powerpc/kvm/book3s_64_mmu_hv.c
+++ b/arch/powerpc/kvm/book3s_64_mmu_hv.c
@@ -109,7 +109,6 @@
 	/* Allocate reverse map array */
 	rev = vmalloc(sizeof(struct revmap_entry) * npte);
 	if (!rev) {
-		pr_err("kvmppc_allocate_hpt: Couldn't alloc reverse map array\n");
 		if (cma)
 			kvm_free_hpt_cma(page, 1 << (order - PAGE_SHIFT));
 		else
